app/controllers/home_controller.rb

```diff
@@ -2,5 +2,6 @@ class HomeController < ApplicationController
   layout 'home'
   def index
     @title = 'Welcome to RecipeApp!'
+    @foods = current_user.foods
```

Comment:
- [ ] @Team Great work implementing the changes requested, however, am getting the error below when I run the application, this is because of how you are fetching foods, if the user is not logged in the current_user provides a nil value causing the error, kindly consider changing line 5 code, below is my suggestion
  `@foods = Food.all`
![image](https://github.com/over-geek/RecipeApp/assets/61408860/7300aee3-71c5-4ea3-99fb-69d5028d93f4)